### PR TITLE
Tg space cleanup imrovements

### PR DESCRIPTION
**Pull request by paha**
- PV size for OS 60G
- Delete PVs that are Bound but older than 3 hours

### CODE REVIEW

**Review by emosbaugh — APPROVED** [review]

File: testgrid/tgrun/pkg/runner/loop.go [hunk 1]
```diff
@@ -175,20 +175,30 @@ func CleanUp() error {
...
 		if pv.ObjectMeta.DeletionTimestamp != nil && time.Since(pv.ObjectMeta.DeletionTimestamp.Time).Hours() > 1 {
 			pv.ObjectMeta.SetFinalizers(nil)
 			p, _ := clientset.CoreV1().PersistentVolumes().Update(ctx(), &pv, metav1.UpdateOptions{})
 
 			fmt.Printf("Removed finalizers on %s, loalPath: %s\n", p.Name, p.Spec.Local.Path)
+			continue
+		}
+
+		// Deleteing stale PVs that are older then 3 hours
+		if time.Since(pv.ObjectMeta.CreationTimestamp.Time).Hours() > 3 && pv.ObjectMeta.DeletionTimestamp == nil {
```
Comment (laverya) [inline testgrid/tgrun/pkg/runner/loop.go:196]: should we do this on the level of pvcs instead?

Comment (paha) [reply testgrid/tgrun/pkg/runner/loop.go:196]: Good point, let me look into it.

File: testgrid/tgrun/pkg/runner/loop.go [hunk 2]
```diff
@@ -175,20 +175,30 @@ func CleanUp() error {
...
+			err = clientset.CoreV1().PersistentVolumes().Delete(ctx(), pv.Name, metav1.DeleteOptions{})
+			if err != nil {
+				fmt.Printf("Failed deleting PV %s\n", pv.Name)
+			}
+			fmt.Println("Deleting PV ", pv.Name)
```
Comment (emosbaugh) [inline testgrid/tgrun/pkg/runner/loop.go:201]: else?

**Comment by emosbaugh** [comment]
closing this as stale

**Pull request closed without merging**